.github/workflows/pg-17-migtests.yml

```diff
@@ -108,27 +110,27 @@ jobs:
           psql "postgresql://yugabyte@yb-tserver-n1:5433/yugabyte" -c "SELECT version();"
 
       - name: "TEST: PG sample schemas (sakila)"
-        if: ${{ !cancelled() && matrix.test_group == 'offline' }}
+        if: ${{ !cancelled() && matrix.test_group == 'offline'}}
```

Comment:
nit: be consistent in formatting here(wrt the spaces being used):
`${{ !cancelled() && matrix.test_group == 'offline' }}`
or
`${{!cancelled() && matrix.test_group == 'offline'}}`